mypy/nodes.py

```diff
@@ -427,17 +427,24 @@ class OverloadedFuncDef(FuncBase, SymbolNode, Statement):
     """
 
     items = None  # type: List[OverloadPart]
+    unanalyzed_items = None  # type: List[OverloadPart]
```

Comment:
I really don't love all these unanalyzed things that aststrip forces upon us. Oh well.

Reply:
I tried few other solutions, but after few attempts to undo the logic of deleting overloads in second pass, I just went with this, although I don't like this either.